lib/models/blueprint.js

```diff
@@ -892,7 +892,7 @@ Blueprint.prototype.addPackageToProject = function(packageName, target) {
 */
 Blueprint.prototype.addPackagesToProject = function(packages) {
   var task         = this.taskFor('npm-install');
-  var install      = (packages.length > 1) ? 'install packages' : 'install package';
+  var installText      = (packages.length > 1) ? 'install packages' : 'install package';
```

Comment:
Can you fix the indentation here?  I personally would prefer a single space, but if we are going to keep the alignment the `=` should be lined up...